### PR TITLE
Updated plugin to use latest syntax

### PR DESCRIPTION
**Pull request by artalat**
Updated plugin to use latest syntax. Added `Switch` component.

### CODE REVIEW

**Comment by artalat** [comment]
@bjavaid please check this PR, I've updated code to support the latest OS but there are many issues. Let's discuss.

File: src/index.tsx [hunk 1]
```diff
@@ -26,53 +23,54 @@ class MaterialUiPlugin extends Plugin {
...
+		'ListItemText': ListItemText,
+		'ListItemRightButton': ListItemRightButton,
+		'Button': Button,
+		'Card': Card,
+		'CardHeader': CardHeader,
+		'CardBody': CardBody,
+		'CardMedia': CardMedia,
+		'CardFooter': CardFooter,
+		'Breadcrumb': Breadcrumb,
+		'SwitchToggle': Switch,
```
Comment (artalat) [inline src/index.tsx]: @bjavaid @amnajunaid this should be deprecated

Comment (amnajunaid) [reply src/index.tsx]: @artalat  Do you mean the set method syntax for registering components ?

Comment (artalat) [reply src/index.tsx]: no, I meant 'SwitchToggle' component. And use 'Switch'. Previously there was a name conflict with the Switch component of router. But that has been renamed to 'RouterSwitch' long ago.

**Pull request merged**